tests/bson/bson-regex-003.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+MongoDB\BSON\Regex #003
+--FILE--
+<?php
+
+require_once __DIR__ . '/../utils/tools.php';
+
+$regexp = new MongoDB\BSON\Regex("regexp");
+printf("Pattern: %s\n", $regexp->getPattern());
+printf("Flags: %s\n", $regexp->getFlags());
+printf("String representation: %s\n", $regexp);
+
+$tests = array(
+    array("regex" => $regexp),
+);
+
+foreach($tests as $n => $test) {
```

Comment:
If you're merging this as-is, don't object if I go back and refactor this at some point. It's dead code that doesn't add any value to the test.